multichain-api-gateway/src/lib.rs

```diff
@@ -45,58 +46,82 @@ impl From<BlockscoutSettings> for ApiEndpoints {
 }
 
 impl ApiEndpoints {
-    async fn make_requests(self, query: &str) -> Vec<(Instance, String)> {
-        let client = Client::new();
+    async fn make_requests(
+        self,
+        query: &str,
+        method: &Method,
+        body: Bytes,
+        headers: &HeaderMap,
+    ) -> Vec<(Instance, String)> {
+        let client = Client::builder()
+            .timeout(std::time::Duration::from_secs(60))
```

Comment:
Default value for a timeout is 1 second. 
[This](https://blockscout.com/eth/mainnet/api-docs#:~:text=curl%20%2Dd%20%27%7B%22addressHash,contract%26action%3Dverify%22) request takes 48 second on my pc